bindings_node/src/inbox_id.rs

```diff
@@ -3,27 +3,44 @@ use napi::bindgen_prelude::Result;
 use napi::bindgen_prelude::Uint8Array;
 use napi_derive::napi;
 use std::sync::Arc;
+use xmtp_api::ApiClientWrapper;
 use xmtp_api_grpc::grpc_api_helper::Client as TonicApiClient;
 use xmtp_common::retry::Retry;
+use xmtp_common::ExponentialBackoff;
 use xmtp_id::associations::generate_inbox_id as xmtp_id_generate_inbox_id;
 use xmtp_id::associations::MemberIdentifier;
-use xmtp_mls::api::ApiClientWrapper;
+use xmtp_proto::api_client::ApiBuilder;
+
+fn retry_strategy() -> Retry<ExponentialBackoff, ExponentialBackoff> {
```

Comment:
Q: so this function set it's own values? regardless of what we set in:
https://github.com/xmtp/libxmtp/pull/1601/files#diff-3a24c3e76565487a710ac9863ac05160128f4f90892e07849b555a6de43a6e8fR174

Reply:
bindings_node/bindings_ffi/bindings_wasm all configure separately. We can expose a function from xmtp_mls to have a unified cooldown strategy and/or just make it a default in the ApiClientWrapper too

Reply:
Yeah, make sense, we can have the default and let each change the configuration based on their prefs.